pkg/config/system_probe.go

```diff
@@ -309,6 +309,7 @@ func InitSystemProbeConfig(cfg Config) {
 	// CWS - Anomaly detection
 	cfg.BindEnvAndSetDefault("runtime_security_config.security_profile.anomaly_detection.event_types", []string{"exec", "dns"})
 	cfg.BindEnvAndSetDefault("runtime_security_config.security_profile.anomaly_detection.minimum_stable_period", 3600)
+	cfg.BindEnvAndSetDefault("runtime_security_config.security_profile.anomaly_detection.workload_warmup_period", 60)
```

Comment:
We believe this will break the new tests added by @spikat . Maybe set this to 0 in the tests ?